src/translate/translate.js

```diff
@@ -1313,6 +1325,20 @@ const nodeQuery = ({
     optimization.earlyOrderBy ? '' : orderByClause
   }${outerSkipLimit}`;
 
+  if (isCount) {
```

Comment:
Should we do `if ... else` and cover the normal case above as well?

Reply:
i did it this way to avoid too many merge conflict with the base since we wont change indentation and all, but it would be cleaner with the `if ...  else`